docs/topics/wasm/wasm-js-interop.md

```diff
@@ -221,6 +221,46 @@ external class User : JsAny {
 }
 ```
 
+### Array interoperability
+
+You can copy JavaScript's `JsArray<T>` into Kotlin's native `Array` or `List` types and vice versa.
```

Comment:
```suggestion
You can convert JavaScript's `JsArray<T>` to Kotlin's native `Array` or `List` types, and likewise you can convert these Kotlin types to `JsArray<T>`.
```

Reply:
Mainly to avoid the Latin idiom for non-English speakers and to avoid repeating "the other way around" from the next line.

Also although this way convert is used multiple times, I feel like "copy" could me misunderstood

Reply:
@skuzmich  Do you think "convert" can work here or would be misunderstood and we prefer "copy"?

Reply:
I'm leaving as "copy" as per suggested by developer.